kubernetes/hands-on-lab/tutorials/node.selector.license.ocishell.md

```diff
@@ -1,14 +1,14 @@
 # Oracle WebLogic Server Kubernetes Operator Tutorial #
 
-### Assign WebLogic pods to licensed nodes ###
+### Assign WebLogic domain to selected nodes ###
 
 This use case is similar to the [Assign WebLogic Pods to Nodes](node.selector.ocishell.md) lab, where individual servers/pods were assigned to specific node(s). However, the focus in this use case on the license coverage.
 
-At v1.13, Kubernetes supports clusters with up to 5000(!) nodes. However, certain software, such as WebLogic Server, requires a license. Using the `nodeSelector` feature, Kubernetes ensures that WebLogic pods end up on licensed worker node(s).
+Starting from v1.13, Kubernetes supports clusters with up to 5000(!) nodes. However, in certain situation you need to deploy WebLogic domain on selected node(s). Using the `nodeSelector` feature, Kubernetes ensures that all WebLogic pods belong to the domain end up on licensed worker node(s).
```

Comment:
Starting from v1.13 -> Starting at v1.13 
in certain situation -> in certain situations
deploy WebLogic domain -> deploy a WebLogic domain
all WebLogic pods belong -> all WebLogic pods that belong